addons/zeus/functions/fnc_ui_teleportPlayers.sqf

```diff
@@ -1,5 +1,5 @@
 /*
- * Author: SilentSpike
+ * Author: SilentSpike, mharis001
  * Initalises the `teleport players` zeus module display
```

Comment:
I know it's not yours but:
`Initalizes the "Teleport Players" Zeus module display.`